File: dungeons-26.1.2/src/main/java/fishmod/features/item/ItemRarity.java
package fishmod.features.item;

/** SkyBlock item rarities and their background tint colors. Ported from blade-addons. */
public enum ItemRarity {
    NONE(0x0),
    COMMON(0xffdddddd),
    UNCOMMON(0xff42ab42),
    RARE(0xff4c4cd0),
    EPIC(0xff671067),
    LEGENDARY(0xffcf8d0a),
    MYTHIC(0xffff55ff),
    DIVINE(0xff4fe4e4),
    SPECIAL(0xffff5555),
    VERY_SPECIAL(0xffc44747),
    ULTIMATE(0xffa10202),
    ADMIN(0xffaa0000);

    private final int color;

    ItemRarity(int color) { this.color = color; }

    public int getColor() { return color; }
}

File: dungeons-1.21.11/src/main/kotlin/fishmod/features/item/ItemRarity.kt
package fishmod.features.item

/** SkyBlock item rarities and their background tint colors. Ported from blade-addons. */
enum class ItemRarity(val color: Int) {
    NONE(0x0),
    COMMON(0xffdddddd.toInt()),
    UNCOMMON(0xff42ab42.toInt()),
    RARE(0xff4c4cd0.toInt()),
    EPIC(0xff671067.toInt()),
    LEGENDARY(0xffcf8d0a.toInt()),
    MYTHIC(0xffff55ff.toInt()),
    DIVINE(0xff4fe4e4.toInt()),
    SPECIAL(0xffff5555.toInt()),
    VERY_SPECIAL(0xffc44747.toInt()),
    ULTIMATE(0xffa10202.toInt()),
    ADMIN(0xffaa0000.toInt())
}
